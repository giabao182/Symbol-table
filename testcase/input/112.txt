INSERT x number
BEGIN
INSERT x string
INSERT y number
BEGIN
INSERT z string
PRINT
BEGIN