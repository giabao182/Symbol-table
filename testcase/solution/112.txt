success
success
success
success
x//1 y//1 z//2
UnclosedBlock: 3