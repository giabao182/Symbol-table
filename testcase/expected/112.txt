success
Redeclared: INSERT x string
success
success
x//0 y//1 z//2
UnclosedBlock: 3